apps/meteor/ee/app/license/server/license.ts

```diff
@@ -49,7 +49,7 @@ class LicenseClass {
 		 * We added a dynamic import here to avoid this issue
```

Comment:
```suggestion
		 * We added a dynamic import here to avoid this issue
		 * @TODO as soon as the Apps-Engine service is available, use it instead of this dynamic import
```